### PR TITLE
Translation of tutorial/part-two

### PR DESCRIPTION
**Pull request by jollypuppet**

### CODE REVIEW

File: docs/tutorial/part-two/index.md [hunk 1]
```diff
@@ -114,43 +114,43 @@ import "./src/styles/global.css"
...
+Esploriamo i **Moduli CSS**. Citando dalla
+[homepage CSS Module](https://github.com/css-modules/css-modules):
 
-> A **CSS Module** is a CSS file in which all class names and animation names
-> are scoped locally by default.
+> Un **Modulo CSS** è un file CSS in cui tutti i nomi delle classi e delle animazioni
+> sono definiti localmente per impostazione predefinita.
 
-CSS Modules are very popular because they let you write CSS normally but with a lot more safety. The tool automatically generates unique class and animation names, so you don't have to worry about selector name collisions.
+I Moduli CSS sono molto popolari perchè permettono di scrivere normalmente CSS ma con molta più sicurezza. Lo strumento genera automaticamente nomi unici di classi e animazioni, in modo da non doverti preoccupare di collisioni di nomi.
```
Comment (f0ffee) [inline docs/tutorial/part-two/index.md]: "perchè" should be "perché" (fussy, I know)

File: docs/tutorial/part-two/index.md [hunk 2]
```diff
@@ -161,9 +161,9 @@ export default ({ children }) => (
 )
 ```
 
-You'll notice you imported a css module file named `container.module.css`. Let's create that file now.
+Noterai che hai importato un file modulo css chiamato `container.module.css`. Adesso creiamo quel file.
```
Comment (f0ffee) [inline docs/tutorial/part-two/index.md]: CSS all caps, let's keep it consistent

File: docs/tutorial/part-two/index.md [hunk 3]
```diff
@@ -190,17 +190,17 @@ export default () => (
...
+Adesso, se visiti `http://localhost:8000/about-css-modules/`, la tua pagina dovrebbe somigliare a questa:
 
-![Page with CSS module styles](css-modules-basic.png)
+![Pagina con stili modulo CSS](css-modules-basic.png)
 
-#### ✋ Style a component using CSS Modules
+#### ✋ Personalizza un componente usando i Moduli CSS
 
-In this section, you'll create a list of people with names, avatars, and short Latin biographies. You'll create a `<User />` component and style that component using a CSS module.
+In questa sezione, creerai una lista di persone con nomi, avatr, e brevi biografie in Latino. Creerai un componente `<User />` e personalizzerai quel componente usando un modulo CSS.
```
Comment (f0ffee) [inline docs/tutorial/part-two/index.md]: "avatr" -> "avatar"; also, language names shouldn't be capitalized in Italian

File: docs/tutorial/part-two/index.md [hunk 4]
```diff
@@ -248,16 +248,15 @@ import Container from "../components/container"
...
 
-![Import result of CSS module in console](css-modules-console.png)
+![Importa il risultato del modulo CSS nella console](css-modules-console.png)
 
-If you compare that to your CSS file, you'll see that each class is now a key in the imported object pointing to a long string e.g. `avatar` points to `src-pages----about-css-modules-module---avatar---2lRF7`. These are the class names CSS Modules generates. They're guaranteed to be unique across your site. And because you have to import them to use the classes, there's never any question about where some CSS is being used.
+Se lo compari al tuo file CSS, vedrai che ogni classe è adesso una chiave nell'oggetto importato puntante ad una lunga stringa, ad esempio `avatar` punta a `src-pages----about-css-modules-module---avatar---2lRF7`. Questi sono i nomi delle classi che i Moduli CSS generano. L'unicità è garantita all'interno del sito. E poichè hai bisogno di importarli per usare le classi, non c'è mai dubbio su dove alcuni CSS vengano usati.
```
Comment (f0ffee) [inline docs/tutorial/part-two/index.md]: More of an opinion than a change request, but I would leave "CSS Modules" as it is.

Also, "poichè" -> "poiché"

File: docs/tutorial/part-two/index.md [hunk 5]
```diff
@@ -298,36 +297,36 @@ export default () => (
 )
 ```
 
-> Tip: Generally, if you use a component in multiple places on a site, it should be in its own module file in the `components` directory. But, if it's used only in one file, create it inline.
+> Suggerimento: Generalmente, se utilizzi un componente di un sito in diversi posti, esso dovrebbe trovarsi nel suo proprio file di modulo nella directory `components`. Ma, se è utilizzato solo in un file, creaolo in linea.
```
Comment (f0ffee) [inline docs/tutorial/part-two/index.md]: "creaolo" -> "crealo"

File: docs/tutorial/part-two/index.md [hunk 6]
```diff
@@ -298,36 +297,36 @@ export default () => (
...
+CSS-in-JS è un approccio di personalizzazione orientato a componenti. Generalmente, è un pattern dove il [CSS è composto inline usando JavaScript](https://reactjs.org/docs/faq-styling.html#what-is-css-in-js).
 
-#### Using CSS-in-JS with Gatsby
+#### Usare CSS-in-JS con Gatsby
 
-There are many different CSS-in-JS libraries and many of them have Gatsby plugins already. We won't cover an example of CSS-in-JS in this initial tutorial, but we encourage you to [explore](/docs/styling/) what the ecosystem has to offer. There are mini-tutorials for two libraries, in particular, [Emotion](/docs/emotion/) and [Styled Components](/docs/styled-components/).
+Ci sono diverse librerie CSS-in-JS e molte presentano già i plugin Gatsby. Non approfondiremo un esempio CSS-in-JS in questo tutorial iniziale, ma ti incoraggiamo a [esplorare](/docs/styling/) cosa l'ecosistema ha da offrire. Ci sono dei mini-tutorial per due librerie, in particolare [Emozione](/docs/emotion/) e [Componenti personalizzati](/docs/styled-components/).
```
Comment (f0ffee) [inline docs/tutorial/part-two/index.md]: Names of libraries shouldn't be translated

File: docs/tutorial/part-two/index.md [hunk 7]
```diff
@@ -298,36 +297,36 @@ export default () => (
...
 
-#### Suggested reading on CSS-in-JS
+#### Letture consigliate su CSS-in-JS
 
-If you're interested in further reading, check out [Christopher "vjeux" Chedeau's 2014 presentation that sparked this movement](https://speakerdeck.com/vjeux/react-css-in-js) as well as [Mark Dalgleish's more recent post "A Unified Styling Language"](https://medium.com/seek-blog/a-unified-styling-language-d0c208de2660).
+Se sei interessato in altre letture, dai un'occhiata a [Christopher "vjeux" Chedeau's 2014 presentation that sparked this movement](https://speakerdeck.com/vjeux/react-css-in-js) così come a [Mark Dalgleish's more recent post "A Unified Styling Language"](https://medium.com/seek-blog/a-unified-styling-language-d0c208de2660).
```
Comment (f0ffee) [inline docs/tutorial/part-two/index.md]: Missing translation

File: docs/tutorial/part-two/index.md [hunk 8]
```diff
@@ -298,36 +297,36 @@ export default () => (
...
 
-### Other CSS options
+### Altre opzioni di CSS
 
-Gatsby supports almost every possible styling option (if there isn't a plugin yet for your favorite CSS option, [please contribute one!](/contributing/how-to-contribute/))
+Gatsby supporta quasi ogni possibile opzione di personalizzazione (Se non è ancora presente un plugin per la tua opzione di CSS preferita, [per favore contribuisci ad inserirla!](/contributing/how-to-contribute/))
```
Comment (f0ffee) [inline docs/tutorial/part-two/index.md]: No capitalization needed inside parenthesis

File: docs/tutorial/part-two/index.md [hunk 9]
```diff
@@ -298,36 +297,36 @@ export default () => (
...
 - [PostCSS](/packages/gatsby-plugin-postcss/)
 
-and more!
+e altre!
 
-## What's coming next?
+## Cosa viene dopo?
 
-Now continue on to [part three of the tutorial](/tutorial/part-three/), where you'll learn about Gatsby plugins and layout components.
+Adesso continua su [terza parte del tutorial](/tutorial/part-three/), dove imparerai riguardo i plugin Gatsby e i componenti di layout.
```
Comment (f0ffee) [inline docs/tutorial/part-two/index.md]: "continua su terza parte" isn't really Italian, maybe something like "continua con la terza parte"?

**Comment by jollypuppet** [comment]
Thanks for the review @f0ffee 👍

File: docs/tutorial/part-two/index.md [hunk 10]
```diff
@@ -248,14 +248,15 @@ import Container from "../components/container"
...
+Se lo compari al tuo file CSS, vedrai che ogni classe è adesso una chiave nell'oggetto importato puntante ad una lunga stringa, ad esempio `avatar` punta a `src-pages----about-css-modules-module---avatar---2lRF7`. Questi sono i nomi delle classi che i CSS Modules generano. L'unicità è garantita all'interno del sito. E poiché hai bisogno di importarli per usare le classi, non c'è mai dubbio su dove alcuni CSS vengano usati.
 
-4. Create a new `<User />` component inline in the `about-css-modules.js` page
-   component. Modify `about-css-modules.js` so it looks like the following:
+4. Crea un componente `User`.
+
+Crea un nuovo componente `<User />` in linea nella pagina componente `about-css-modules.js`. Modifica `about-css-modules.js` in modo da farlo somigliare al seguente codice:
```
Comment (adnsio) [inline docs/tutorial/part-two/index.md]: Can we delete "Crea un componente `User`." on line 257 and start with line 259?

**Pull request merged**